### PR TITLE
don't truncate channels name

### PR DESCRIPTION
**Pull request by lcoenengithub**
**What is this?**

This removes `no-wrap` constraints around channel title per #1391 

**Illustration**

image

### CODE REVIEW

**Comment by rafalpawlow** [comment]
@lcoenengithub base branch is wrong, should be `dev`

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts [hunk 1]
```diff
@@ -58,9 +58,6 @@ export const TitleContainer = styled.div`
 
 export const Title = styled(Text)`
   margin-bottom: 0;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-  overflow: hidden;
   max-width: 600px;
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts]: The designer said that there should be no max-width here, so you should remove this line. You'll need to update also the container grid to make this work if I'm not mistaken.

File: packages/atlas/src/components/_channel/ChannelWithVideos/ChannelWithVideos.styles.ts [hunk 2]
```diff
@@ -27,9 +27,6 @@ export const InfoWrapper = styled.div`
 
 export const ChannelTitle = styled(Text)`
   max-width: 200px;
```
Comment (drillprop) [inline packages/atlas/src/components/_channel/ChannelWithVideos/ChannelWithVideos.styles.ts]: same as above

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts (hunk 1 above)
Comment (lcoenengithub) [reply packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts]: Thanks! Fixed

**Review by kdembler — CHANGES_REQUESTED** [review]
Channel name on channel page doesn't wrap and overflows the layout:
Screenshot 2022-02-21 at 09 11 56

**Comment by lcoenengithub** [comment]
@kdembler I modified the grid layout to make it wrap again. Thanks for your feedback!

image

**Review by kdembler — APPROVED** [review]

**Review by drillprop — CHANGES_REQUESTED** [review]
...

Edit: made a mistake - ignore this comment for now

File: packages/atlas/src/components/_channel/ChannelWithVideos/ChannelWithVideos.styles.ts [hunk 3]
```diff
@@ -6,6 +6,10 @@ import { Text } from '@/components/Text'
 import { Button } from '@/components/_buttons/Button'
 import { sizes } from '@/styles'
 
+export const Title = styled(Text)`
+  margin-right: ${sizes(6)};
+`
```
Comment (drillprop) [inline packages/atlas/src/components/_channel/ChannelWithVideos/ChannelWithVideos.styles.ts]: Last thing. Why do we need this margin? I think we could safely remove this.

**Review by drillprop — CHANGES_REQUESTED** [review]

**Review by drillprop — APPROVED** [review]

**Pull request merged**